src/processor/Package.ts

```diff
@@ -330,4 +331,63 @@ export class Package {
       pullAt(sd.rules, rulesToRemove);
     });
   }
+
+  private removeDateRules(): void {
+    let allDatesMatch = true;
+    let date: string;
+    [...this.profiles, ...this.extensions, ...this.valueSets, ...this.codeSystems].forEach(sd => {
+      for (const rule of sd.rules) {
+        if (!allDatesMatch) break; // If any date hasn't matched the others, stop looking at other rules
+        if (
+          rule instanceof ExportableCaretValueRule &&
+          rule.caretPath === 'date' &&
+          rule.path === ''
+        ) {
+          const dateValue = rule.value as string; // If the rule assigns a date, the value will be a string.
+          if (!date) date = dateValue; // Set the date to match
+          if (date !== dateValue) {
+            allDatesMatch = false;
+          }
+        }
+      }
+    });
+
+    // If all dates are the same, are defined to the second, and are in GMT,
+    // we can assume they were set by the IG Publisher and can be safely removed.
+    const dateTimeRegex = /^([0-9]([0-9]([0-9][1-9]|[1-9]0)|[1-9]00)|[1-9]000)(-(0[1-9]|1[0-2])(-(0[1-9]|[1-2][0-9]|3[0-1])(T([01][0-9]|2[0-3]):[0-5][0-9]:([0-5][0-9]|60)(\.[0-9]+)?(Z|(\+|-)((0[0-9]|1[0-3]):[0-5][0-9]|14:00)))?)?)?$/;
+    const dateTimeMatch = date?.match(dateTimeRegex);
+    const usesGMT = dateTimeMatch?.[12] === '+00:00'; // dateTimeMatch[12] will hold timezone info if present
+    if (allDatesMatch && usesGMT) {
+      const DEFAULT_DATE = new ExportableCaretValueRule('');
```

Comment:
It freaks me out a little to all-caps this, because that makes me thing is is a global variable. But that's more personal preference so if you like the caps, keep the caps.

Reply:
I think of CAPS as being a true constant -- i.e., a value _defined_ at compile time (as opposed to a `constant` that is _assigned_ some value at runtime).

Reply:
Oh.  But based on my definition, I guess that also means it should not be CAPS since we don't know `DEFAULT.value` until runtime. ;-)

Reply:
Oh dear I'm sorry to freak you out. I can definitely change the caps, but I also now realize I did something similar in `removeDefaultSlicingRules`, so I'm not sure if it makes you more or less comfortable to have both of them caps or just one of the caps. Or neither of them caps.

Reply:
Oh yeah.  Well crap.  I forgot about path.  In that case, I've probably also violated my own definition of all CAPS constants.  Nick, why did you have to say anything?!  Now look what you've done!

Reply:
All caps are fine, I'm not too picky.